### ISSUE TITLE
Still alive???

### ISSUE THREAD

**Issue by NCC1701M** [sass/libsass-net#70]
On you [site](https://sass-lang.com/libsass) you wrote:

> libsass-net is updated regularly, and is probably the best bet.

I'd like to disagree. There is only one release and this was created in 2015.

**Comment by darrenkopp** [comment]
You are correct. In defense of that statement I'm sure it was last modified when libsass-net was being actively. However, libsass-net has not been updated since a point where I was not able to reconcile changes made in libsass because it was beyond my limited understanding on how to write the P/Invoke calls.

[LibSassHost](https://github.com/Taritsyn/LibSassHost) is an up-to-date wrapper for libsass, however.
